opal/mca/btl/usnic/btl_usnic_module.c

```diff
@@ -1783,6 +1806,18 @@ static int init_one_channel(opal_btl_usnic_module_t *module,
         goto error;
     }
 
+    /* Ensure that we got a CQ that is at least as long as we asked
+       for */
+    if (cq_attr.size < cq_num) {
+        opal_show_help("help-mpi-btl-usnic.txt",
+                       "internal error during init",
+                       true,
+                       opal_process_info.nodename,
+                       module->linux_device_name,
+                       "created CQ is too small", __FILE__, __LINE__);
```

Comment:
Missing rc and fi_strerror() args.  Also same with show_help, above, after fi_cq_open().